rollup.config.js

```diff
@@ -53,6 +53,9 @@ export default {
     replace({
       __GOOGLE_ANALYTICS_ID__:
         process.env.CONTEXT === "production" && process.env.GOOGLE_ANALYTICS_ID,
+      __GLEAN_VERSION__: execSync("git rev-list HEAD --max-count=1")
```

Comment:
I think just `__VERSION__` is better here, `__GLEAN_VERSION__` might imply that the version has something to do with the [Glean SDK](https://github.com/mozilla/glean) (which it does not)

```suggestion
      __VERSION__: execSync("git rev-list HEAD --max-count=1")
```